packages/table/test/gridTests.ts

```diff
@@ -158,6 +159,15 @@ describe("Grid", () => {
         });
     });
 
+    describe("style", () => {
+        it("returns display: none if region not inside grid", () => {
+            const grid = new Grid(test7s, test13s);
+            const region = Regions.cell(5, 5, 10, 10);
+            const regionStyle = grid.getRegionStyle(region);
+            expect(regionStyle.display).to.equal("none");
+        });
```

Comment:
Can we add a test—or at least an `expect`—for every relevant region cardinality (`CELLS`, `FULL_COLUMNS`, `FULL_ROWS`) to maximize confidence and test coverage?